src/fixedSizeSquareMatrixOpsImpl.hpp

```diff
@@ -88,7 +88,18 @@ struct SquareMatrixOps< 2 >
   static auto determinant( MATRIX const & matrix )
   {
     checkSizes< 2, 2 >( matrix );
-    return matrix[ 0 ][ 0 ] * matrix[ 1 ][ 1 ] - matrix[ 0 ][ 1 ] * matrix[ 1 ][ 0 ];
+
+    // Aliases for matrix [[a, b],[c, d]] for improved readability
+    auto const & a = matrix[0][0];
```

Comment:
I think `matrix[ 0 ][ 0 ]` is more readable.

Reply:
OK, I will change this.

Reply:
> Also this modification makes calculating the determinant significantly more costly. I'm not sure you can answer this question but maybe the kind of matrices that pop up in the unit tests aren't representative of the normal use case.

It is 3 flop vs 2 flop. Look at the second variant I listed in https://godbolt.org/z/4bGdjYbd5

I don't think this is a big deal...for 3d it may be a bit more costly...but not much. I don't think we will often run into this sort of round off problem, but then again, if it doesn't cost much to "fix", then I kind of like this approach.